tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -126,130 +149,206 @@ xdescribe('Effective Date Time component', () => {
 
-    const validEvent = getLastEvent(wrapper, 'valid')
+    // set everything except hour
+    await wrapper.find('#date-text-field').setValue(wrapper.vm.dateToDateString(today))
+    wrapper.vm.$refs.minuteSelector.setValue(today.getMinutes().toString())
+    await wrapper.find('#period-selector').setValue(today.getHours() >= 12 ? 'pm' : 'am')
 
-    // Verify the Valid emit event is false at this point
-    expect(validEvent).toEqual(true)
+    // wait a bit for validation to complete
+    await flushPromises()
+
+    // Verify the last Valid event is false
+    expect(getLastEvent(wrapper, 'valid')).toEqual(false)
   })
 
-  it('emits a invalid state when the Future Effective is selected and DateTime is invalid', async () => {
-    const wrapper = wrapperFactory({ EffectiveDateTime: dateTimeInvalid })
+  it('emits a invalid state when Future Effective is selected and no minute is entered', async () => {
+    const wrapper = wrapperFactory({
+      currentJsDate: today,
+      effectiveDateTime: dateTimeDefault
+    })
 
+    // select Future Effective
     const radioInput = wrapper.findAll('input[type="radio"]')
     const radioIsFutureEffective = radioInput.at(1)
     await radioIsFutureEffective.trigger('click')
 
-    const invalidEvent = getLastEvent(wrapper, 'valid')
+    // set everything except minute
+    await wrapper.find('#date-text-field').setValue(wrapper.vm.dateToDateString(today))
+    wrapper.vm.$refs.hourSelector.setValue((today.getHours() % 12).toString())
+    await wrapper.find('#period-selector').setValue(today.getHours() >= 12 ? 'pm' : 'am')
+
+    // wait a bit for validation to complete
+    await flushPromises()
 
-    // Verify the Valid emit event is false at this point
-    expect(invalidEvent).toEqual(false) // *** BROKEN
+    // Verify the last Valid event is false
+    expect(getLastEvent(wrapper, 'valid')).toEqual(false)
   })
 
-  it('displays an invalid Date Alert when the Date is invalid', async () => {
-    const wrapper = wrapperFactory({ EffectiveDateTime: dateTimeInvalid })
+  it('emits a valid state when Future Effective is selected and valid date and time are entered', async () => {
+    const wrapper = wrapperFactory({
+      currentJsDate: today,
+      effectiveDateTime: dateTimeDefault
+    })
 
+    // select Future Effective
     const radioInput = wrapper.findAll('input[type="radio"]')
     const radioIsFutureEffective = radioInput.at(1)
     await radioIsFutureEffective.trigger('click')
 
-    const minDate = wrapper.vm.formatDateString(wrapper.vm.minDate)
-    const maxDate = wrapper.vm.formatDateString((wrapper.vm.maxDate))
+    // set everything
+    await wrapper.find('#date-text-field').setValue(wrapper.vm.dateToDateString(today))
+    wrapper.vm.$refs.hourSelector.setValue((today.getHours() % 12).toString())
+    wrapper.vm.$refs.minuteSelector.setValue(today.getMinutes().toString())
+    await wrapper.find('#period-selector').setValue(today.getHours() >= 12 ? 'pm' : 'am')
```

Comment:
So this killed a lot of time unfortunately...
1. you can use `setValue()` on a v-text-field
2. I could find no information (not even in Vuetify's own Jest tests -- they're disabled!) how to set the value of a v-combobox; this was the only way
3. you can use `setValue()` on a v-select

I believe this is the first time we have tried to set a v-combobox's value.

Reply:
Crazy, a gap in the vue-test-utils?! 
It's finicky stuff either way, to bypass in an instance like this it must be fine to just set/emit the value manually? ie, set it to store or a local prop, whatever you are trying to test.

Reply:
Haha, that was my last resort but I didn't have to do it after all.